platforms/core-runtime/base-services/build.gradle.kts

```diff
@@ -74,6 +60,22 @@ packageCycles {
 }
 
 jmh.includes = listOf("HashingAlgorithmsBenchmark")
+
 tasks.isolatedProjectsIntegTest {
     enabled = false
 }
+
+// TODO: Why is base-services responsible for this?
```

Comment:
@jvandort AFAICT introduced in https://github.com/gradle/gradle/commit/ba7596186caea80ac56883c33c634332cc7c10b4

> The build receipt creation is now handled by the baseServices project,
as that is the place, where it is packaged in the end.